src/index.js

```diff
@@ -22,6 +22,10 @@ ReactDOM.render(
             <Route path='/r/:subreddit/comments/:threadID/:junk/:commentID' component={Thread} />
             <Route path='/r/:subreddit/comments/:threadID' component={Thread} />
             <Route path='/r/:subreddit' component={Subreddit} />
+            <Redirect from='/user/:username/comments/:threadID/:junk/:commentID'
+                      to='/r/u_username/comments/:threadID/:junk/:commentID' />
```

Comment:
Looks like you're missing the colon between `u_` and `username` here. It should be: `to='/r/u_:username/comments/:threadID/:junk/:commentID'

Reply:
Yep, you're right! Thanks!